Client/RecommendationClient.php

```diff
@@ -22,4 +22,13 @@ public function updateContent($contentId);
      * @return void
      */
     public function deleteContent($contentId);
+
+    /**
+     * @param int $userId YooChoose user ID
+     * @param int $scenarioId YooChoose scenario ID
+     * @param int $locationId location ID
+     * @param int $limit maximum limit of fetched recommendations
+     * @return json|null
+     */
+    public function getRecommendations( $userId, $scenarioId, $locationId, $limit );
```

Comment:
I'm thinking this is not the best place for this method...

I know it's called Client, but there was no question of fetching recos when it was named :-)

My main concern is that updating the Recommendation Server and fetching Recommendations don't have much to do with each other. Both HTTP services even have different hostnames. Furthermore, since the Client was implemented using Notifications, you end up adding real time recommendation fetching in a class named YooChooseNotifier. It doesn't sound right.

I suggest we add a new interface for fetching recommendations, and rename this one to make it clear it takes care of updating the recommendation server. What do you think ?